### DISCUSSION TITLE
Is it possible to generate images using only model.pt file without converting to ckpt file?

### DISCUSSION THREAD

**Discussion by healthyfat** [bmaltais/kohya_ss#77] [Q&A]
At the moment to check the result of Lora training I convert created model.pt file into model.ckpt and then run "gen_img_diffusers" script to generate sample images. Is there a way to do it without the need to convert model?

**Comment by iznanka** [comment]
Use auto1111 and install the extension https://github.com/kohya-ss/sd-webui-additional-networks. You can apply the trained Lora to any model

**Comment by healthyfat** [comment]
Thanks. I am well aware you can do that in Auto1111, but I am interested in Kohya specifically.
